Formula/pgcli.rb

```diff
@@ -67,6 +69,9 @@ class Pgcli < Formula
   end
 
   def install
+    # Append Homebrew's bin directory to the PATH so Psycopg can find the pg_config binary.
+    ENV.append_path "PATH", "#{HOMEBREW_PREFIX}/bin"
```

Comment:
Homebrew removes it from the path before install. Then, it manually adds each dependency's binaries to the path. But, since we don't have the dependency, we need it in the path here.

It is not a permanent addition to the path -- just for that instance.